include/boost/numeric/ublas/tensor/fwd.hpp

```diff
@@ -0,0 +1,199 @@
+//
+//  Copyright (c) 2018-2019, Cem Bassoy, cem.bassoy@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Fraunhofer IOSB, Ettlingen, Germany
+//
+
+#ifndef _BOOST_NUMERIC_UBLAS_TENSOR_FWD_HPP_
+#define _BOOST_NUMERIC_UBLAS_TENSOR_FWD_HPP_
+
+#include <iostream>
```

Comment:
This header shouldn't be needed.

Reply:
I will remove it.

Reply:
resolving conversation as file is outdated.